hbase-server/src/test/java/org/apache/hadoop/hbase/quotas/TestSpaceQuotaBasicFunctioning.java

```diff
@@ -221,4 +228,32 @@ public void testTableQuotaOverridesNamespaceQuota() throws Exception {
         Bytes.toBytes("reject"));
     helper.verifyViolation(policy, tn, p);
   }
+
+  @Test
+  public void testDisablePolicyQuotaAndViolate() throws Exception {
+    TableName tableName = helper.createTable();
+    helper.setQuotaLimit(tableName, SpaceViolationPolicy.DISABLE, 1L);
+    helper.writeData(tableName, SpaceQuotaHelperForTests.ONE_MEGABYTE * 2L);
+    TEST_UTIL.getConfiguration()
+        .setLong("hbase.master.quotas.region.report.retention.millis", 100);
+
+    HMaster master = TEST_UTIL.getMiniHBaseCluster().getMaster();
+    MasterQuotaManager quotaManager = master.getMasterQuotaManager();
+
+    // Make sure the master has report for the table.
+    Waiter.waitFor(TEST_UTIL.getConfiguration(), 30 * 1000, new Waiter.Predicate<Exception>() {
+      @Override
+      public boolean evaluate() throws Exception {
+        Map<RegionInfo, Long> regionSizes = quotaManager.snapshotRegionSizes();
+        List<RegionInfo> tableRegions =
+            MetaTableAccessor.getTableRegions(TEST_UTIL.getConnection(), tableName);
+        return regionSizes.containsKey(tableRegions.get(0));
+      }
+    });
+
+    // Check if disabled table region report present in the map after retention period expired.
+    // It should be present after retention period expired.
+    Assert.assertTrue(quotaManager.snapshotRegionSizes().keySet().stream()
```

Comment:
Can you pull the lambda out of the `assertTrue` to make this a little more readable? e.g.
```
final int regionSizes = quotaManager.snapshotRegionSizes()...;
Assert.assertTrue(regionSizes > 0);
```

Reply:
done